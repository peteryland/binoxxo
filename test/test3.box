o6o2
5o3x
4o
2xx
10
2x1o
x2o2oo1x
x6o
9x
2o1o
