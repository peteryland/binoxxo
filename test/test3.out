O______O__
_____O___X
____O_____
__XX______
__________
__X_O_____
X__O__OO_X
X______O__
_________X
__O_O_____

OXOXXOXOXO
OXOOXOXXOX
XOXOOXOXOX
XOXXOXOOXO
OXOXXOXXOO
OXXOOXOXOX
XOXOOXOOXX
XOOXXOXOXO
OXXOXOOXOX
XOOXOXXOXO

